en/flight_controller/pixhawk4_mini.md

```diff
@@ -0,0 +1,113 @@
+# Pixhawk 4 Mini
+
+The *Pixhawk<sup>&reg;</sup> 4 Mini* autopilot is designed for engineers and hobbyists who are looking to tap into the power of *Pixhawk 4* but are working with smaller drones. *Pixhawk 4 Mini* takes the FMU processor and memory resources from the *Pixhawk 4* while eliminating normally unused interfaces. This allows the *Pixhawk 4 Mini* to be small enough to fit in a 250mm racer drone.
```

Comment:
Shouldn't the (R) be after "Mini"? Or is it only Pixhawk that is registered?

Reply:
"while eliminating normally unused interfaces." to "while eliminating interfaces that are normally unused."

Reply:
I believe on pixhawk. Because that is how it is on the logo.